app/tests/spec/lib/fxa-client.js

```diff
@@ -795,6 +839,20 @@ define(function (require, exports, module) {
             assert.isTrue(sessionData.verified);
           });
       });
+
+      it('passes along an optional `metricsContext`', function () {
+        var metricsContext = {};
+        return client.completePasswordReset(email, password, token, code, relier, { metricsContext: metricsContext})
```

Comment:
Can you break up this line, or use object shorthand for the `metricsContext` portion so that it's limited to 100-120 chars?